src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -58,6 +58,42 @@
 _MemoryBIO = collections.namedtuple("_MemoryBIO", ["bio", "char_ptr"])
 
 
+def _encode_name_constraints(backend, name_constraints):
```

Comment:
Sure. (This module is pretty unwieldy so we should probably move this stuff to another module, but that should be in another PR)